File: backend/Domain/Entities/Organization.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace Domain.Entities
{
    public class Organization : BaseLogDomain
    {
        public int Id { get; set; }
        public string? Name { get; set; }
        public string? Address { get; set; }
        public string? Director { get; set; }
        public string? Nomer { get; set; }
        public string? Pin { get; set; }
        public string? Okpo { get; set; }
        public int? OrganizationTypeId { get; set; }
        public string? OrganizationTypeName { get; set; }
        public string? PostalCode { get; set; }
        public string? Ugns { get; set; }
        public string? RegNumber { get; set; }
    }
}

File: frontend/src/constants/Organization.ts
import { Dayjs } from "dayjs";

export type Organization = {
  id: number;
  name: string;
  address: string;
  director: string;
  nomer: string;
  pin: string;
  okpo: string;
  postal_code: string;
  ugns: string;
  reg_number: string;
  organization_type_id: number;
  organization_type_name?: string;
  created_at: Dayjs;
  updated_at: Dayjs;
  created_by: number;
  updated_by: number;
};

export type OrganizationCreateModel = {
  id: number;
  name: string;
  address: string;
  director: string;
  nomer: string;
  pin: string;
  okpo: string;
  postal_code: string;
  ugns: string;
  reg_number: string;
  organization_type_id: number;
};